validation/resource-config_test.go

```diff
@@ -0,0 +1,93 @@
+package validation
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-service/retry"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func TestResourceConfig(t *testing.T) {
+	isExcluded := map[uint64]bool{
+		888:          true, // OP_Labs_chaosnet_0
+		997:          true, // OP_Labs_devnet_0
+		11155421:     true, // OP_Labs_Sepolia_devnet_0
+		11763071:     true, // Base_devnet_0
+		129831238013: true, // Conduit_devnet_0
+	}
+
+	checkResourceConfig := func(t *testing.T, chain *ChainConfig) {
+		rpcEndpoint := Superchains[chain.Superchain].Config.L1.PublicRPC
+
+		require.NotEmpty(t, rpcEndpoint)
+
+		client, err := ethclient.Dial(rpcEndpoint)
+		require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+
+		contractAddress, err := Addresses[chain.ChainID].AddressFor("SystemConfigProxy")
+		require.NoError(t, err)
+
+		uint128Max, ok := new(big.Int).SetString("ffffffffffffffffffffffffffffffff", 16)
+		if !ok {
+			panic("cannot construct uint128Max")
+		}
+		desiredResourceConfig := bindings.ResourceMeteringResourceConfig{
+			MaxResourceLimit:            20000000,
+			ElasticityMultiplier:        10,
+			BaseFeeMaxChangeDenominator: 8,
+			MinimumBaseFee:              1000000000,
+			SystemTxMaxGas:              1000000,
+			MaximumBaseFee:              uint128Max,
+		} // from OP Mainnet
+
+		actualResourceConfig, err := getResourceConfigWithRetries(context.Background(), common.Address(contractAddress), client)
+		require.NoErrorf(t, err, "RPC endpoint %s: %s", rpcEndpoint)
+
+		require.Equal(t, desiredResourceConfig, actualResourceConfig, "resource config unacceptable")
+
+		t.Logf("resource metering acceptable")
```

Comment:
I was doing that previously, and can easily put it back in. However, since this is now run as a named subtest (L64), this line (and the others you flagged) will actually log the message with the chain identifier:

```
=== RUN   TestResourceConfig/Base_devnet_0_(11763072)
    resource-config_test.go:58: resource metering acceptable
```

Reply:
Oh ok, never mind then!